### DISCUSSION TITLE
Support request for LiliGO T-Display S3 AMOLED

### DISCUSSION THREAD

**Discussion by jpenninkhof** [arendst/Tasmota#19204] [Support request]
The LiLiGO T-Display-S3 is currently supported, but has seen an upgrade to an AMOLED version. Along with the upgrade of the display from LCD to AMOLED, they have also changed the (default) interface from SPI to QSPI. It is supposed to work faster tha SPI, which is welcome given its higher resolution display. However, QSPI is currently not supported by Tasmota yet. Needless to say that it would be great to have Tasmota to support this upgraded board, including QSPI as well.

- https://www.lilygo.cc/products/t-display-s3-amoled
- https://github.com/Xinyuan-LilyGO/T-Display-S3-AMOLED

I've raised this question on Discord first. @Jason2866 suggested to tag @gemu2015, being the "Display witcher" :)

**Comment by gemu2015** [comment]
supporting QSPI would be a bit of work. i would suggest to jumper the board to SPI and try to adapt that with the standard SPI universal display interface.

since i do not own such a board and will not buy one in the near future i will not adapt it myself.

**Comment by karoCB** [comment]
umgelötet auf SPI ist schnell - aber welche Verbindungen sind jetzt bei Tasmota einzustellen ?
![Pinout_T-Display-S3 AMOLED](https://github.com/user-attachments/assets/c89c318f-4782-49b1-8956-14a64e4adf5f)

**Reply by sfromis** [reply]
By now, Tasmota supports both octal and quad SPI, if you select a binary to match. Next issue would be getting the display supported, I'm not aware of anyone having done the `display.ini` setup for RM67162.
